File: Java/MenstrualCycle/MenstruationCycleMain.java
// MenstruationCycleMain.java

import java.util.Scanner;
public class MenstruationCycleMain {
     static Scanner input = new Scanner(System.in);
    
public static void main(String[] args) {
int userInput;
do {
    System.out.println("\n===========================================");
System.out.println("      Welcome to Menstruation Cycle Tracker");
System.out.println("===========================================");
System.out.println("  1. Add User Details");
System.out.println("  2. View Menstrual Summary");
System.out.println("  3. Update Menstruation Cycle");
System.out.println("  4. Exit");
System.out.println("===========================================");
System.out.print("Please enter your choice (1-4): ");
    userInput = input.nextInt();
    input.nextLine();

    switch(userInput) {
        case 1 -> MenstruationCycle.addUserDetails();
        case 2 -> MenstruationCycle.viewMenstrualSummary();
        case 3 -> MenstruationCycle.updateMenstruationCycle();
        case 4 -> System.out.println("GoodBye!! Have a nice day!");
        default -> System.out.println("Invalid Option. Please try again ");
    }
} 
while (userInput != 4);
}
}

File: daySeven/python_menstruation_cycle/menstruation_cycle_main.py
# menstruation_cycle_main.py
import menstruation_cycle

def menstruation_cycle_main():
    while True:
        print("\n===========================================")
        print("      Welcome to Menstruation Cycle Tracker")
        print("===========================================")
        print("  1. Add User Details")
        print("  2. View Menstrual Summary")
        print("  3. Update Menstruation Cycle")
        print("  4. Exit")
        print("===========================================")
        choice = input("Please enter your choice (1-4): ").strip()

        if choice == "1":
            menstruation_cycle.add_user_details()
        elif choice == "2":
            menstruation_cycle.view_menstrual_summary()
        elif choice == "3":
            menstruation_cycle.update_menstruation_cycle()
        elif choice == "4":
            print("GoodBye!! Have a nice day!")
            break
        else:
            print("Invalid Option. Please try again.")

if __name__ == "__main__":
    menstruation_cycle_main()
